NSSNamespace/User_Authentication
# removed superfluous comments, deleted mysterious unneccessary class f…

## Changelog
- Removed unnecessary and mysterious ```Class.cs``` file on root directory
- Removed unused, commented out code
- Added ```[Authorize]``` decoration to appropriate classes and methods

## To Test
- ```git fetch```
- ```git checkout ew-commentsAndAuth```
- Check ```Order```, ```Payment```, and ```Products``` controllers, make sure all classes and methods that require authorization are properly decorated
